### DISCUSSION TITLE
My Paladin just do Templeruns

### DISCUSSION THREAD

**Discussion by Wullff91** [isid0re/SoloLeveling#201] [Q&A]
Hey my paladin just doing now 24 h templeruns, is there any bug or issu?
He doesnt do act 4 & act 5 until now and is lvl 34.

**Comment by isid0re** [comment]
Thanks. I've been sorting thru a few bugs and this one is new. I'll look into it.

**Reply by Wullff91** [reply]
ok thanks

**Reply by Wullff91** [reply]
20:27:28 (SCL-PAL) Window has unexpectedly exited... starting profile
20:27:41 (SCL-PAL) Restarting
after finishing the rune he always get this error

**Reply by isid0re** [reply]
Pushed the templerun fix. Please update

**Reply by Wullff91** [reply]
ok looks like its going on 
starting a sorc lvl 33 and just finished act 3 after the fix
